### ISSUE TITLE
Installer step 2 gives a 500 error 

### ISSUE THREAD

**Issue by AdmiraalA** [nZEDb/nZEDb#2470]
###Expected behaviour

Step 2 would create database (Ubuntu 16.04.2/Percona 5.7 and git and composer setup) 
###Actual behaviour
Uncaught ErrorException: Couldn't change session tz_session! in /var/www/nZEDb/nzedb/db/DB.php:1375
Stack trace:
#0 /var/www/nZEDb/nzedb/db/DB.php(212): nzedb\db\DB->connect(Array)
#1 /var/www/nZEDb/www/install/step2.php(104): nzedb\db\DB->__construct(Array)
#2 {main}
  thrown in /var/www/nZEDb/nzedb/db/DB.php on line 1375" while reading response header from upstream, client: xx.xx, server: nzedb.evilness.nl, request: "POST /install/step2.php?

**Comment by Wally73** [comment]
***NOTE*** If you have not already done so, you MUST populate the database's timezone tables
     as described in their documentation. For Debian/Ubuntu users 'mysql_tzinfo_to_sql
     /usr/share/zoneinfo | mysql -u root -p mysql' should load them easily, this will require
     restarting the database server.

mariadb 10.x.x is the preferred DB (and the only one supported)

**Comment by AdmiraalA** [comment]
Thanx for the advice , this fixed the 500 error :)
Now i'm running into the error that 32 tables are being created and @ the install page i see : 
Unable to create database and data. Check permissions of your mysql user.
File and reload grants setup correctly.

**Comment by KillerdogKD** [comment]
Hi,

i have the same problem in step2 👍 
`Fatal  error: Uncaught exception 'ErrorException' with message 'Couldn't  change session tz_session!' in C:\WPN-XM\www\nzedb\nzedb\db\DB.php on  line 1375--
`

i'm on windows with WPN-XM, but have the same things with wamp, xamp or wtserver.

I configure php.ini with "date.timezone = Europe/Berlin" but it does the same.

What should i do to populate mariaDB with correct timezone ?

Thanks

**Comment by niel** [comment]
@KillerdogKD, no idea. Window's is not officially supported, you would be better off installing in an Ubuntu virtual machine on Windows. Or if you are using Win10, install the Ubuntu app when it becomes available ;-)

**Comment by KillerdogKD** [comment]
thanks for answer.
 
i try un ubuntu (14.04 - 16.04 - 16.10) with virtualbox, but facing different problem 
- Package error on install
- White page on localhost/nzedb/install at step2
- Error connecting to mysql

i follow different tutorials but i'm blocked at step2 everytime. it just run me crazy :)

anyway, thanks for your answer

**Comment by niel** [comment]
We have guides for installing on Ubuntu (unfortunately not one specific to 16.04 or later yet). If you follow one of those you can get specific support in our IRC channel when you run into problems.

**Comment by brinkdogg** [comment]
@KillerdogKD I have the exact same issue - This is crazy 
HA HA HA HA!!! after the tables were created using the Time fix I kept getting 
The following error(s) were encountered:
• Unable to create database and data. Check permissions of your mysql user.

So for the hell of it I just changed /install/step2.php to /install/step3.php and it moved me forward!! but then I hit a wall at step6. http 500 and everything is dead... (bangheadwall)
